lib/engine/corporation.rb

```diff
@@ -52,6 +52,7 @@ def initialize(sym:, name:, **opts)
       @float_percent = opts[:float_percent] || 60
       @floated = false
       @max_ownership_percent = opts[:max_ownership_percent] || 60
+      @hold_above_max_ownership_percent_ok = opts[:hold_above_max_ownership_percent_ok] || false
```

Comment:
name kinda long

Reply:
I agree. I went for clarity over length. Any suggestions? can_hold_above_max maybe?

Reply:
yea
sounds goods